profiles/coreos/base/use.mask

```diff
@@ -4,6 +4,12 @@ kdbus
 # We default to python 3.6 for now
 python_targets_python3_7
 python_single_target_python3_7
+python_targets_python3_8
```

Comment:
Maybe we can reword the commit to explain why we mask this python versions:

From the PR body:
> [...] it is enough to modify the python-utils-r1.eclass to add the missing python versions (3.8, 3.9 and 3.10), and mask them.

Reply:
Right, I have added some more words to commit messages in both PRs.